src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -257,8 +257,6 @@ private void InitDiscovery()
             _api.LogManager, _api.Timestamper, _api.CryptoRandom,
             _api.NodeStatsManager, _api.IpResolver
         );
-
-        _api.DiscoveryApp.Initialize(_api.NodeKey.PublicKey);
```

Comment:
This is moved somewhere esle?

Reply:
Moved to CompositeDiscoveryApp's constructor.